roles/postgresql/tasks/main.yml

```diff
@@ -145,26 +199,55 @@
   when:
     - postgres_host is not none
     - postgresql_is_local
+    - ansible_os_family == "Debian"
     - running_on_server
   become: true
   notify:
     - reload local postgres
 
-- include_tasks: pg_hba.yml
-- include_tasks: warm_standby_leader.yml
+- name: PostgreSQL | ensure access to postgres server (RedHat)
+  ansible.builtin.blockinfile:
+    path: '/var/lib/pgsql/data/pg_hba.conf'
+    insertafter: "# Put your actual configuration here"
+    block: |
+      # localhost
+  when:
+    - postgres_host is not none
+    - postgresql_is_local
+    - ansible_os_family == "RedHat"
+    - running_on_server
+  become: true
+  notify:
+    - restart local postgres
+
+- name: PostgreSQL | configure access to postgres server
+  ansible.builtin.include_tasks: pg_hba.yml
```

Comment:
How is this different from the task at the bottom of this file? Are we creating the pg_hba.yml list of allowed servers twice?